.github/workflows/cf2pulumi-release.yml

```diff
@@ -1,39 +1,48 @@
-name: CF2Pulumi Release
-"on":
+name: cf2pulumi-release
+on:
   push:
     tags:
-      - v*.*.*
-      - '!v*.*.*-**'
-      
+    - v*.*.*
+    - "!v*.*.*-**"
 env:
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-
+  GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+  PROVIDER: aws-native
+  PULUMI_ACCESS_TOKEN: ${{ secrets.PULUMI_ACCESS_TOKEN }}
+  PULUMI_LOCAL_NUGET: ${{ github.workspace }}/nuget
+  NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+  NODE_AUTH_TOKEN: ${{ secrets.NPM_TOKEN }}
+  NUGET_PUBLISH_KEY: ${{ secrets.NUGET_PUBLISH_KEY }}
+  PYPI_PASSWORD: ${{ secrets.PYPI_PASSWORD }}
+  TRAVIS_OS_NAME: linux
+  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }}
+  PULUMI_GO_DEP_ROOT: ${{ github.workspace }}/..
+  AWS_REGION: us-west-2
+  PULUMI_API: https://api.pulumi-staging.io
 jobs:
   release:
     runs-on: macos-latest
     steps:
-      - name: Checkout
-        uses: actions/checkout@v2
-      - name: Unshallow clone
-        run: git fetch --prune --unshallow
-      - name: Install pulumictl
-        uses: jaxxstorm/action-install-gh-release@v1.1.0
-        with:
-          repo: pulumi/pulumictl
-      - name: Install Go 1.17.x
-        uses: actions/setup-go@v2
-        with:
-          go-version: '1.17.x'
-      - name: Run GoReleaser
-        uses: goreleaser/goreleaser-action@v2
-        with:
-          version: latest
-          args: -p 1 -f .goreleaser.cf2pulumi.yml release --rm-dist --timeout 60m0s
-        env:
-          GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
-      - name: Chocolatey Package Deployment
-        run: |
-          CURRENT_TAG=v$(pulumictl get version --language generic -o)
-          pulumictl create choco-deploy -a cf2pulumi ${CURRENT_TAG}
-        env:
-          GITHUB_TOKEN: ${{ secrets.PULUMI_BOT_TOKEN }}
+    - name: Checkout Repo
+      uses: actions/checkout@v2
+      with:
+        lfs: true
+    - name: Unshallow clone for tags
+      run: git fetch --prune --unshallow --tags
+    - name: Install pulumictl
+      uses: jaxxstorm/action-install-gh-release@v1.5.0
+      with:
+        repo: pulumi/pulumictl
+    - name: Install Go
```

Comment:
Just confirming that we're bumping to 1.18?

Reply:
Yep, this is intentional!